### PR TITLE
Re-enable Apple.com.xml

### PR DESCRIPTION
**Pull request by koops76**

### CODE REVIEW

**Comment by cschanaj** [comment]
Fixed 1326fbefbfeea3a8230558739bcc44c1390d0ccc. Thanks a lot!!

File: src/chrome/content/rules/Apple.com.xml [hunk 1]
```diff
@@ -36,6 +36,7 @@
 		- init.ess	(503, akamai)
 		- userprofile.itunes *
 		- rss.lists	(shows lists; mismatched, CN: lists.apple.com)
+		- rss.itunes (redirects to HTTP)
```
Comment (J0WI) [inline src/chrome/content/rules/Apple.com.xml]: I'm not able to reproduce:

Comment (koops76) [inline src/chrome/content/rules/Apple.com.xml]: Try using `curl`.

Comment (J0WI) [inline src/chrome/content/rules/Apple.com.xml]: Yes, I used curl.

Comment (koops76) [inline src/chrome/content/rules/Apple.com.xml]: @J0WI They have fixed it it seems.

Comment (koops76) [inline src/chrome/content/rules/Apple.com.xml]: @J0WI Done.

**Comment by J0WI** [comment]
Oh, Travis has a different behavior for some reason.

**Comment by koops76** [comment]
@J0WI Fixed.

**Pull request merged**